Sources/FacebookCoreTests/AccessTokenWalletTests.swift

```diff
@@ -63,6 +63,13 @@ class AccessTokenWalletTests: XCTestCase {
     )
   }
 
+  func testSharedWallet() {
+    let sharedWallet = AccessTokenWallet.shared
+
+    XCTAssertTrue(sharedWallet === AccessTokenWallet.shared,
```

Comment:
Would `XCTAssertEqualObjects` check reference?

```suggestion
    XCTAssertEqualObjects(sharedWallet, AccessTokenWallet.shared,
```

Reply:
Unfortunately I don't think this is available in swift. :(